build.gradle

```diff
@@ -24,16 +30,79 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
     antlr 'org.antlr:antlr4:4.5'
 
-    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
-    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    // using groovy 2.3 to avoid problems in eclipse
+    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
```

Comment:
Staying on topic, my intent is to make the project approachable a easy to use by anyone. That's why I decided to downgrade groovy to avoid problems with other IDEs.
Groovy is only used for spock tests, so I don't feel we are missing anything critical using 2.3 instead of 2.4.
If you prefer, I can remove the comment to avoid controversies.